feat(extension): Rename Extension installer buttons for clarity

acastrodev opened the pull request:
# PR Details

   - Update launcher extension button labels
   - New labels: "Visual Studio Shader {0} Extensions"
   - Add link under `VisualStudioExtension` label: "An installed Visual Studio is required, which can be found here"

## Description

This commit addresses the issue of misleading launcher extension button labels, making it clearer that users are installing an extension, not the IDE itself. The new labels and added clarification help redirect users to the Visual Studio download page to avoid confusion.

## Related Issue

Resolves: #1773

## Motivation and Context

Improves Launcher UX
![image](https://github.com/stride3d/stride/assets/6575712/0f7a555d-022c-4d51-b2bf-5c6eb8129311)

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Doprez commented:
did you add the link under "here"? It doesnt seem highlighted in the screenshot.

acastrodev commented:
> did you add the link under "here"? It doesnt seem highlighted in the screenshot.

I added "An installed Visual Studio is required, which can be found here" entirely

Doprez commented:
> > did you add the link under "here"? It doesnt seem highlighted in the screenshot.
> 
> I added "An installed Visual Studio is required, which can be found here" entirely

Right, but is "Which can be found here" referring to the extension or the full IDE?

acastrodev commented:
> > > did you add the link under "here"? It doesnt seem highlighted in the screenshot.
> > 
> > 
> > I added "An installed Visual Studio is required, which can be found here" entirely
> 
> Right, but is "Which can be found here" referring to the extension or the full IDE?

To the full IDE, also added the link tooltip: 
![image](https://github.com/stride3d/stride/assets/6575712/dc1f814f-9704-4195-8e00-4663080b78dd)
